proto/minder/v1/minder.proto

```diff
@@ -1128,6 +1149,45 @@ message DeleteRuleTypeResponse {
 
 // --- END RuleType CRUD definitions
 
+// --- START EvaluationResults definitions
+
+message ListEvaluationResultsRequest{
+    // context is the context in which the evaluation results are evaluated.
+    Context context = 1;
+
+    // profile is the profile name or ID to retrieve results for.
+    // If empty, return evaluation results for profiles selected by
+    // an empty label_filter.
+    oneof profile_selector {
+        // ID can contain either a profile name or an ID
+        string profile = 2;
+        // Filter profiles to only those matching the specified labels.
+        // 
+        // The default is to return all user-created profiles; the string "*" can
+        // be used to select all profiles, including system profiles.  This syntax
+        // may be expanded in the future.
+        string label_filter = 3;
+    }
+
+    // If set, only return evaluation results for the named rules.
+    // If empty, return evaluation results for all rules
```

Comment:
I guess it's not intentional but the comment here is the same as the next one

Reply:
Oops, thanks, fixed!